x-pack/plugin/sql/sql-cli/src/main/java/org/elasticsearch/xpack/sql/cli/Cli.java

```diff
@@ -84,7 +84,12 @@ private static void configureJLineLogging() {
      * Build the CLI.
      */
     public Cli(CliTerminal cliTerminal) {
-        super("Elasticsearch SQL CLI");
+        // rewire / replace LoggingAwareCommand / CommandLoggingConfigurator::configureLoggingWithoutConfig
```

Comment:
I don't see the usefulness of this comment in the code itself. Would have been more helpful in the PR description, though together with the mention of not depending on `sql-action` (which pulls `core` lib and `x-content`) but on the more trimmed down `sql-proto`.